roles/keypair/defaults/main.yaml

```diff
@@ -0,0 +1,10 @@
+---
+prefix: test- # This should be overridden in inventory
+
+keypair_name: "{{ (prefix + 'KeyPair') }}"
+keypair_private_key_dest: "{{ ('~/.ssh/' + keypair_name + '.pem') }}"
+
+# set this to force key deletion
+force_delete_key: false
```

Comment:
true?

Reply:
copied that from ansible-role-vpc where default is not to force deletion. Here it make sense to set it other way around